packages/volto/news/6591.fix

```diff
@@ -0,0 +1 @@
+- Added useState to hangle siteLogo
```

Comment:
See https://6.docs.plone.org/contributing/index.html#change-log-entry for correct file extension and content guidance.